carma_planning_msgs/srv/GetEmergencyRoute.srv

```diff
@@ -0,0 +1,20 @@
+#
+# GetEmergencyRoute.srv
+# 
+# Provided by the EmergencyResponseVehicle node. Call the service to load a predefined 
+# route. Responds with success boolean and name of loaded route
```

Comment:
A call to this service is used to determine whether a route has already been pre-loaded into the EmergencyResponseVehicle node, so let's consider changing the wording here from "to load a predefined route" to "to receive the name of the node's current emergency route, if it was successfully loaded" or something similar.